UnitsNet.NanoFramework/GeneratedCode/Frequency/UnitsNet.NanoFramework.Frequency.nuspec

```diff
@@ -2,14 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>UnitsNet.nanoFramework.Frequency</id>
-    <version>4.89.0</version>
+    <version>4.90.0</version>
     <title>Units.NET Frequency - nanoFramework</title>
     <authors>Andreas Gullberg Larsen, nanoFramework project contributors</authors>
     <owners>UnitsNet</owners>
     <license type="expression">MIT</license>
     <projectUrl>https://github.com/angularsen/UnitsNet</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Adds Frequency units for Units.NET on .NET nanoFramework.&#10; For .NET or .NET Core, use UnitsNet instead.</description>
+    <description>Adds Frequency units for Units.NET on .NET nanoFramework.&#10;For .NET or .NET Core, use UnitsNet instead.</description>
```

Comment:
Is this `&#10;` character intentional? Why not a regular space? I pushed a commit removing it, but let me know if that was a mistake.